cypress/integration/user.spec.ts

```diff
@@ -22,7 +22,8 @@ context('User Integration Test', () => {
     cy.createPin()
     cy.get('#menu-team').click()
     cy.get('#locationSearchInput').type('Mkushi')
-    cy.get('#locationOptionc59c440f-beaa-4d8d-93f3-a4fb21a72979').click()
+    //cy.get('#locationOptionc59c440f-beaa-4d8d-93f3-a4fb21a72979').click()
+    cy.get('#locationOption121b9d2e-a43b-4d3e-9ba5-fd985d2de34b').click()
```

Comment:
@azantor64 this is technical debt that we need to resolve.  We cannot have this button ID depending on a unique ID for a location.  These IDs change regularly.  We need a new way to identify this location in the select but I dont know how.  We should discuss